src/preamble.js

```diff
@@ -2032,4 +2032,343 @@ Module['FS_createPreloadedFile'] = FS.createPreloadedFile;
 var cyberDWARFFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
 #endif
 
+#if BINARYEN
+function integrateWasmJS(Module) {
```

Comment:
OK, great, this LGTM then. Also, (just so I understand) before this was at the very top of the JS file, and now it's in the preamble that gets pasted after environment setup but before everything else, right?